burnman/tools.py

```diff
@@ -643,3 +644,109 @@ def check_eos_consistency(m, P=1.e9, T=300., tol=0.01, verbose=False):
             print('Not satisfied all EoS consistency constraints for {0:s}'.format(m.to_string()))
             
     return consistency
+
+
+def _pad_ndarray_inverse_mirror(array, padding):
```

Comment:
Well, that wasn't well formatted. Starting array:

| 1 | 2 | 2 | 1 |
| 1 | 3 | 3 | 1 |
| 1 | 4 | 4 | 1 |
| 1 | 5 | 5 | 1 |

After first step (note the use of the corner values)

| - | 7/4  | 7/4  | - |
| - | 10/4 | 10/4 | - |
| - | 13/4 | 13/4 | - |
| - | 16/4 | 16/4 | - |